app/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/repository/meldekort/MeldeperiodePostgresRepo.kt

```diff
@@ -82,6 +90,26 @@ internal class MeldeperiodePostgresRepo(
         }
     }
 
+    override fun hentSakerSomMåGenerereMeldeperioderFra(ikkeGenererEtter: LocalDate): List<Sak> {
+        return sessionFactory.withSessionContext { sessionContext ->
+            sessionContext.withSession { session ->
+                session.run(
+                    queryOf(
+                        """
+                            with temp as (
+                                select s.*, max(m.til_og_med) as til_og_med from sak s join meldeperiode m on s.id = m.sak_id
```

Comment:
```suggestion
                                select s.id, s.siste_dag_som_gir_rett, max(m.til_og_med) as til_og_med from sak s join meldeperiode m on s.id = m.sak_id
```
Kan optimalisere litt 🤷